public/scripts/gif.js

```diff
@@ -0,0 +1,65 @@
+var imagesData = [];
+var imageLinks = [];
+
+function getImageArray(data){
+
+	imagesData = [];
+	$("#gif #gifContent #gifApp").empty();
+	$.each(data, function (index, obj) {
+		if($(obj.event_html).attr("src").indexOf("notcaptured") < 0){
+		imagesData.push(obj);
+		}
+	});
+
+	for(var i = 0; i<imagesData.length; i++)
+	{
+		imageLinks[i] = $(imagesData[i].event_html).attr('src');
+		console.log(imageLinks[i]);
+	}
+
+	var interval = document.getElementById("interval").value;
+
+	create(imageLinks, interval);
+
+	document.getElementById("gifButton").addEventListener("click", updateGif);
+}
+
+function updateGif(){
+	$("#gifApp").empty();
+	var interval = document.getElementById("interval").value;
+	create(imageLinks, interval);

```

Comment:
```suggestion
	createGif(imageLinks, interval);
```

Since the function name is changed below.